src/export/CodeSystemExporter.ts

```diff
@@ -54,10 +71,58 @@ export class CodeSystemExporter {
     }
   }
 
+  private setCodeCaretRules(
+    codeSystem: CodeSystem,
+    csStructureDefinition: StructureDefinition,
+    rules: CodeCaretValueRule[]
+  ) {
+    resolveSoftIndexing(rules);
```

Comment:
What soft indices might exist that need to be resolved?  We currently don't support soft indices on caret rules, and I don't think concept paths can hold soft indices either.

Or is this forward-looking so it is already in place for if/when we do support soft indices on caret paths?

Reply:
I probably put this line in when initially writing the function as a mirror of what happens in `setCaretRules`, but it isn't needed right now, so there's no reason for it to stay.